pom.xml

```diff
@@ -335,6 +335,16 @@
         <enabled>true</enabled>
       </snapshots>
     </pluginRepository>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+      <id>ebr</id>
+      <url>https://repo.eclipse.org/content/repositories/ebr/</url>
```

Comment:
LOCUS-39 does not explain why we need this added ;)

are these plugins not mirrored to central or similar ?